include/qbsp/qbsp.hh

```diff
@@ -153,9 +153,10 @@ typedef struct mtexinfo_s {
         if (this->miptex > other.miptex)
             return false;
         
-        if (this->flags < other.flags)
+        // CHECK: is it worth sorting by all members?
```

Comment:
Yeah, I think so, because this operator< defines equality for these mtexinfo_t structs when used as map keys.

If it doesn't compare all members, we could have a case you call `FindTexinfo()`, pass in some texinfo, but get back a false match (e.g. one that has phong enabled when it was requested to be off).

I might try this trick of using std::tuple to help implement the comparison function:
https://en.cppreference.com/w/cpp/utility/tuple/tie